pkg/trace/operation.go

```diff
@@ -169,7 +177,7 @@ func FromContext(ctx context.Context) (Operation, error) {
 	case operation:
 		o.operation = val
 	default:
-		return Operation{}, fmt.Errorf("not an Operation")
+		return Operation{Context: context.Background()}, fmt.Errorf("not an Operation")
```

Comment:
I would rather this be empty since it's an error condition.   I also don't see an instance where it's necessary.